### ISSUE TITLE
Why are most generated mask images blurry?

### ISSUE THREAD

**Issue by zw-12** [tianrun-chen/SAM-Adapter-PyTorch#46]
hi，
after fine-tuning on the Kvasir dataset, many of the mask images generated on the Kvasir dataset images were very blurry.
![cju2rnkt22xep0801as160g9t](https://github.com/tianrun-chen/SAM-Adapter-PyTorch/assets/71587035/7851ea46-3ec0-4498-85ba-9efc3ef7f48b)
![cju2nd7l7z98o0799gfjvyfmw](https://github.com/tianrun-chen/SAM-Adapter-PyTorch/assets/71587035/48c569e5-c2e2-461c-8001-bd5c96c32fb0)

and after fine-tuning on our custom dataset, the correct mask was also not generated

![0-0-inp](https://github.com/tianrun-chen/SAM-Adapter-PyTorch/assets/71587035/baae30fa-1e2f-408e-a932-4c1d18a6d113)

**Comment by tianrun-chen** [comment]
how many epoches have you trained? if the model is not well trained, it becomes blurry

**Comment by ybk66** [comment]
我训练后的效果跟你的很像。mask由形似斑点的像素组成。请问你解决了吗？我有一个想法是关于再次后处理mask的。
![dayimask](https://github.com/tianrun-chen/SAM-Adapter-PyTorch/assets/107694755/39404840-288c-43cf-bcd2-c73dcc395c55)
